pkg/syncer/syncer_test.go

```diff
@@ -841,3 +853,27 @@ func getPodSpec(namespace string, labels map[string]string, pvcName string, phas
 func waitForListerSync() {
 	time.Sleep(1 * time.Second)
 }
+
+func GetContainerOrchestratorInterface(orchestratorType int) (commonco.COCommonInterface, error) {
+	fakeK8SCO := &FakeK8SOrchestrator{
+		featureStates: map[string]string{
+			"volume-extend": "true",
+			"volume-health": "true",
+			"csi-migration": "false",
+		},
+	}
+	return fakeK8SCO, nil
+}
+
+func (c *FakeK8SOrchestrator) IsFSSEnabled(ctx context.Context, featureName string) bool {
```

Comment:
Can you add func doc for exported function? otherwise linter check will fail.

Reply:
Done